key_test.go

```diff
@@ -59,6 +59,13 @@ func Test_Key(t *testing.T) {
 			}
 		})
 
+		Convey("Get parent-keys that are available to the child section", func() {
+			parentKeys := cfg.Section("package.sub").ParentKeys()
+			for _, k := range parentKeys {
+				So(k.Name(), ShouldEqual, fmt.Sprintf("%s", "CLONE_URL"))
```

Comment:
I think you can replace `fmt.Sprintf("%s", "CLONE_URL")` with `"CLONE_URL"`, plain text string does not need format.